plugins/pipelines/templates/updateTrusted.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const metaSchema = schema.models.templateMeta.base;
+
+module.exports = () => ({
+    method: 'PUT',
+    path: '/pipeline/templates/{namespace}/{name}/trusted',
+    options: {
+        description: "Update a pipeline template's trusted property",
+        notes: 'Returns null if successful',
+        tags: ['api', 'pipeline', 'templates', 'trusted'],
+        auth: {
+            strategies: ['token'],
+            scope: ['admin', '!guest']
+        },
+
+        handler: async (request, h) => {
+            const { namespace, name } = request.params;
+            const { pipelineTemplateFactory } = request.server.app;
+            const { trusted } = request.payload;
+
+            const pipelineTemplateMeta = await pipelineTemplateFactory.get({
+                name,
+                namespace
+            });
+
+            if (!pipelineTemplateMeta) {
+                throw boom.notFound(`Pipeline template ${namespace}/${name} does not exist`);
+            }
+
+            if (!trusted) {
```

Comment:
if no `version` is specified then use the latest, else update with the version from payload

Reply:
resolving this as per discussion in slack